build/profiles/freenas/ports-system.pyd

```diff
@@ -310,7 +310,7 @@ ports += {
         "OPTIONS_FILE_SET+=PAM_SMBPASS",
         "OPTIONS_FILE_SET+=PTHREADPOOL",
         "OPTIONS_FILE_SET+=QUOTAS",
-        "OPTIONS_FILE_UNSET+=SPOTLIGHT",
+        "OPTIONS_FILE_SET+=SPOTLIGHT",
```

Comment:
Doesnt this bring a lot of dependencies?

Reply:
In this case added dependencies are devel/bison, textproc/flex. Samba 4.12 introduced alternative elasticsearch spotlight backend, which doesn't require bringing in tracker.